src/runner.js

```diff
@@ -132,8 +132,12 @@ assign(Runner.prototype, {
     return queryPromise
       .then((resp) => {
         const processedResponse = this.client.processResponse(resp, runner);
+        let queryContext;
+        if (this.builder && isFunction(this.builder.queryContext)) {
```

Comment:
Is there some case when `this.builder.queryContext` is not a Function? Or when this.builder is not declared? If those checks are here only because some tests are not initialized with builder, then tests should be changed instead (for example by mocking builder).

Reply:
Yeah there were some failing tests and I assumed it meant that some code would also fail. I'll remove the checks and go about fixing those tests :)